src/main/commands/scan.yml

```diff
@@ -1,13 +1,17 @@
 description: Detect bugs and vulnerabilities
 parameters:
   sonar_token_variable_name:
-    description: the name of the environment variable where the SonarCloud API token is stored 
+    description: the name of the environment variable where the SonarCloud API token is stored
     default: SONAR_TOKEN
     type: env_var_name
   cache_version:
     description: increment this value if the cache is corrupted and you want to start with a clean cache
     default: 1
     type: integer
+  executor_os:
+    description: specify the OS of the executor. Supported options are linux and macosx.
```

Comment:
We probably want to be specific about this because this orb probably would not work on a Windows executor without some more fundamental changes.